Python/DZ/DZ1/dz2.py

```diff
@@ -0,0 +1,21 @@
+# Задача 4:
+# Петя, Катя и Сережа делают из бумаги журавликов.
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок,
+# если известно, что Петя и Сережа сделали одинаковое количество журавликов,
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+#
+# *Пример:*
+#
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+
+def count_crane():
+    a = int(input("Введите количество журавликов "))
+    a = a / 3
+    value = f'{int(a/2)} {int(a*2)} {int(a/2)}'
```

Comment:
Можно значительно упростить код.
```suggestion
    a = int(input("Введите количество журавликов ")) // 6
    value = f'{a} {a * 4} {a}'
```